src/gluonts/distribution/gaussian.py

```diff
@@ -102,6 +102,20 @@ def s(mu: Tensor, sigma: Tensor) -> Tensor:
             s, mu=self.mu, sigma=self.sigma, num_samples=num_samples
         )
 
+    def quantile(self, level: Tensor) -> Tensor:
+        F = self.F
+        # we consider level to be an independent axis and so expand it
+        # to shape (num_levels, 1, 1, ...)
```

Comment:
For example, this expansion here is a bit counterintuitive if one reasons in terms of numpy’s broadcasting

Reply:
But yeah, I guess we should be looking at how mxnet does broadcasting :-)

Reply:
Yeah, we may need to iterate a bit on this. For now I assume the simplest case where level is a 1d array and the method returns a tensor with `(num_levels, ...DISTRIBUTION SHAPE...)`. I documented this in the base class.

Reply:
makes sense, thanks